ROX-19928: Updates to policy after 4.9 gui changes (#99954)
openshift/openshift-docs: modules/preview-policy-violations.adoc

= Reviewing the policy and previewing violations

[role="_abstract"]
When creating a policy, {product-title-short} provides a preview window where you can view your policy configuration and preview the violations that will occur if the policy is implemented.

.Procedure

. Verify that the policy configuration is configured with the correct options.

> . View the results in the *Preview violations* panel to ensure that the policy is working. This panel provides additional information, including if build phase or deploy phase deployments have violations of the policy.

Reviewer: ```suggestion
. View the results in the *Preview violations* panel to ensure that the policy is working. This panel provides additional information, including whether build phase or deploy phase deployments have policy violations.
```
